content/docs/user-guide/assets/pipeline/product-assets.md

```diff
@@ -0,0 +1,51 @@
+---
+linkTitle: Product Assets 
+title: Product Assets
+description: Product assets are the runtime ready form of content in O3DE projects.
+weight: 400
+toc: true
+---
+
+A **product asset** is the runtime ready form of content in O3DE projects.
+
+The [Asset Processor](/docs/user-guide/assets/assets/asset-processor/) uses [Asset Builders](/docs/user-guide/assets/pipeline/asset-builders/) to generate product assets from [source assets](/docs/user-guide/assets/pipeline/source-assets/).
+
+This page covers best practices for generating product assets from an asset builder.
+
+# Determistic Generation
+
+It's best practice to make product asset generation deterministic. The same source asset, with the same version of the asset builder for that source asset, should generate the exact same product asset each time it is generated. Both locally for the same contributor, and for all contributors on an O3DE project. This should be done for several reasons.
+
+* It's easier to debug issues with product assets if they are identical for all contributors on a project.
```

Comment:
Focus this on leading with the reason first ie

1) Asset ID stablility - 
2) Debugging - its easier
3) Performance - Job dependencies
4) Bundled asset tracking consistency - ...